excercise-1/src/main/java/com/piseth/java/school/excercise1/service/impl/SchoolServiceImpl.java

```diff
@@ -82,6 +82,7 @@ public Optional<Teacher> findOldestTeacher(List<ClassRoom> classRooms, int start
 
 	@Override
 	public List<Teacher> findTeacherByNumberOfStudent(List<ClassRoom> classRooms, int startYear, int numberOfStudent) {
+		// this is filtered by number of student per class & year, while I think, we should filter by year of combine class of each teacher
```

Comment:
![image](https://user-images.githubusercontent.com/112937826/215385538-1c0f2a06-9943-41df-a7d9-812f0d403b58.png)

But based on this structure how do you think?